### DISCUSSION TITLE
LOOK AT THE DISCUSSION TAB -------------------^^^^^^^^

### DISCUSSION THREAD

**Discussion by MomboMan** [fullstackhero/blazor-starter-kit#344] [General]
Hi, I'm not an admin so if you want to tell me to shut up that would be warranted.

I'd like to suggest that all questions / discussions go under the newly made Discussions tab.  I'm seeing lots of questions that I would think make the guys trying to squash bugs crazy.  If you have a question or would like to feel good about yourself and help someone then go look under the discussions tab.  This really works for the MudBlazor group.

If you have a bug or feature request (which should really be moved elsewhere) then place it here.

My $0.02

Craig

**Comment by dotnetshadow** [comment]
I agree, perhaps we need a sticky note to give some guidance on where to place stuff

One of the issues is when you create a **New Issue** you are prompted with the following screen
![image](https://user-images.githubusercontent.com/296247/136487985-224b6f5e-edb4-40f1-83cc-141fa3b3760e.png)

Therefore people (including myself) tend to put Feature Requests and Documentation requests in issues.
